torch/csrc/jit/codegen/fuser/cpu/fused_kernel.cpp

```diff
@@ -59,52 +59,52 @@ static bool programExists(const std::string& program) {
 }
 
 #ifdef _MSC_VER
-c10::optional<std::string> exec(const std::string& cmd) {
-  std::array<char, 128> buffer;
-  std::string result;
+c10::optional<std::wstring> exec(const std::wstring& cmd) {
+  std::array<wchar_t, 128> buffer;
+  std::wstring result;
   std::unique_ptr<FILE, decltype(&_pclose)> pipe(
-      _popen(cmd.c_str(), "r"), _pclose);
+      _wpopen(cmd.c_str(), L"r"), _pclose);
   if (!pipe) {
     return c10::nullopt;
   }
-  while (fgets(buffer.data(), static_cast<int>(buffer.size()), pipe.get()) !=
+  while (fgetws(buffer.data(), static_cast<int>(buffer.size()), pipe.get()) !=
          nullptr) {
     result += buffer.data();
   }
   return result;
 }
 
-inline std::string& rtrim(std::string& s, const char* t = " \t\n\r\f\v") {
+inline std::wstring& rtrim(std::wstring& s, const wchar_t* t = L" \t\n\r\f\v") {
   s.erase(s.find_last_not_of(t) + 1);
   return s;
 }
 
 void activate() {
-  char* root = nullptr;
-  std::string cmd;
-  c10::optional<std::string> exec_out;
-  std::string path;
-  std::string vcruntime_plat;
-  std::string envvars;
+  wchar_t* root = nullptr;
+  std::wstring cmd;
+  c10::optional<std::wstring> exec_out;
+  std::wstring path;
+  std::wstring vcruntime_plat;
+  std::wstring envvars;
 
   // Checking whether the environment is already activated
-  if (getenv("VSCMD_ARG_TGT_ARCH")) {
+  if (_wgetenv(L"VSCMD_ARG_TGT_ARCH")) {
```

Comment:
not necessary?

Reply:
Yeah I just like to make the `W` usage unified in this method.